### PR TITLE
fix plot that misses one timestep

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[[EBV Analyzer] Plot zeigt einen Zeitschritt zu wenig](https://trello.com/c/gUOwvf0e/1139-ebv-analyzer-plot-zeigt-einen-zeitschritt-zu-wenig)

File: projects/dashboards/ebv-analyzer/src/app/ebv-selector/ebv-selector.component.ts [hunk 1]
```diff
@@ -194,12 +198,23 @@ export class EbvSelectorComponent implements OnInit, OnDestroy {
...
                         of(sessionToken),
                     ]),
                 ),
                 first(),
-                mergeMap(([workflowId, selectedCountry, sessionToken]) => {
+                mergeMap(([workflowId, rasterWorkflowMetaData, selectedCountry, sessionToken]) => {
+                    let spatialResolution: [number, number] = [0.1, 0.1];
+                    if (
+                        rasterWorkflowMetaData.resolution &&
+                        rasterWorkflowMetaData.resolution.x > 0.1 &&
```
Comment (jdroenner) [inline projects/dashboards/ebv-analyzer/src/app/ebv-selector/ebv-selector.component.ts:211]: I din't think this is an issue here but 0.1° is ~160km if the layer is in EPSG:4326. If it is in UTM then it is 0.1 m. So i think we should add some more logic here. There is no projection injection in the metadata request so the SRS may be heterogenous. We could request the unit of the SRS and check if it is m or ° and than decide that...

Comment (ChristianBeilschmidt) [reply projects/dashboards/ebv-analyzer/src/app/ebv-selector/ebv-selector.component.ts]: [e422207](https://github.com/geo-engine/geoengine-ui/pull/545/commits/e42220712b3f2ecd1cccab63a3b9a4d32e884301)

**Review by jdroenner — APPROVED** [review]

**Comment by jdroenner** [comment]
bors merge

**Pull request merged**